docs/using.rst

```diff
@@ -94,6 +94,10 @@ Operating System Packages
   * Port: ``cd /usr/ports/security/py-letsencrypt && make install clean``
   * Package: ``pkg install py27-letsencrypt``
 
+**Arch Linux**
+
+  * Packages: ``pacman -S letsencrypt letsencrypt-nginx letsencrypt-apache letshelp-letsencrypt``
```

Comment:
1. Can you use `.. code-block:: shell` as in other places of the documentation?
2. "Packages" should be omitted IMHO
3. Have you actually tested nginx and apache plugins? We officially support only Apache, and only on (some) Debian based distros; there are known problems in other distros...
4. Yup, lines should be limited to 79 chars, and 72 for long blocks of text. Use `\` to indicate wrapped line.